README.adoc

```diff
@@ -1,14 +1,14 @@
 ifdef::env-github[]
 :status:
-:outfilesuffix: .adoc
-:caution-caption: :fire:
-:important-caption: :exclamation:
-:note-caption: :paperclip:
 :tip-caption: :bulb:
+:note-caption: :information_source:
```

Comment:
@camilamacedo86 I noticed that you changed some of the values here. I also used these values in the UI README, should we update those also for consistency?

Reply:
I changed the images because they are wrong. I added the same images that are been in used in the inter8ly. Note that before this change when we use a NOTE it was shown a clip which is an attachment.

Reply:
@camilamacedo86 Could you update them in the README for the UI as part of your follow on pr?